front/style/pages/_grande-marche-europe.scss

```diff
@@ -0,0 +1,287 @@
+    height: 100vh;
+    width: 100vw;
+    background-color: $purple;
+    z-index: 15;
+    animation-timing-function: ease-in-out;
+    transition: all 2s;
+    .logo {
+        top: 50%;
+        transform: translateY(-50%);
+        z-index: 4;
+        position: relative;
+        font-size: 4em;
+        font-weight: bold;
+        opacity: 0;
+        transition: all 1s;
+    }
+}
+
+
+.grande-marche-europe {
+    .filter {
+        background-color: #000;
+        height: 101%;
+        width: 100%;
+        opacity: 0.2;
+        position: absolute;
+        top: 0;
+    }
+    /*VIDEO*/
+    .wrapper {
+        width: 100%;
+        height: 100vh;
+    }
+    .wrapper video {
+        object-fit: cover;
+        width: 100%;
+        height: 100%;
+    }
+    video::-webkit-media-controls {
+        display: none !important;
+    }
+    /*HEADER*/
+    header {
+        display: flex;
+        justify-content: space-between;
+        position: absolute;
+        top: 0;
+        width: 100%;
+        z-index: 3;
+    }
+    .logo {
+        padding: 2em;
+    }
+    .socials {
+        width: 15em;
+        display: flex;
+        justify-content: space-around;
+        padding: 2em;
+        line-height: 1em;
+        a {
+            color: #FFF;
+            border: none;
+            &:hover {
+                color: $purple;
+            }
+        }
+    }
+    /*TITLE*/
+    h1 {
+        color: white;
+        margin: 0;
+        position: absolute;
+        z-index: 2;
+        font-size: 3em;
+        top: 20%;
+        line-height: 1.5em;
+        transform: translateY(-50%);
+        width: 100%;
+        text-align: center;
+        @media (max-width: $tablet) {
+            font-size: 2em;
+        }
+        @media (max-width: $mobile) {
+            font-size: 2em;
+            line-height: 1.5em;
+            margin-top: 2em;
+        }
+    }
+
+    p {
+        font-family: sans-serif;
+        line-height: 1.7;
+        color: #4A4A4A;
+        margin: 20px;
+    }
+    /*COUNTER*/
+    .counter {
+        color: white;
+        margin: 0;
+        position: absolute;
+        z-index: 2;
+        top: 50%;
+        transform: translateY(-50%);
+        width: 100%;
+        text-align: center;
+        h2 {
+            font-size: 8em;
+            font-weight: bold;
+        }
+    }
+    .counter-unit {
+        display: flex;
+        justify-content: space-between;
+        align-items: baseline;
+        width: 50%;
+        margin: 0 auto;
+        @media (max-width: $tablet) {
+            justify-content: space-evenly;
+            width: 100%;
+        }
+
+        p {
+            color: #FFF;
+            text-transform: uppercase;
+            font-size: 1.4em;
+            font-weight: 100;
+            margin: 0;
+            left: 5px;
+            position: relative;
+            @media (max-width: $mobile) {
+                font-size: 0.6em;
+            }
+        }
+        span {
```

Comment:
Missing line break. Generally let's add spaces between classes, or not if you thing it's better long term. :)